src/index.js

```diff
@@ -444,7 +453,7 @@ export default class extends PureComponent {
       const pointer = this.lazy.getPointerCoordinates();
       const brush = this.lazy.getBrushCoordinates();
 
-      this.drawInterface(this.ctx.interface, pointer, brush);
+      // this.drawInterface(this.ctx.interface, pointer, brush);
```

Comment:
```suggestion
      this.drawInterface(this.ctx.interface, pointer, brush);
```

Please uncomment this line again, otherwise the catenary and draw position is not shown.

Reply:
I'm not able to add these changes since it's not my work. You could just make a commit after you've merged this. I have opened a pr for the readme: https://github.com/embiem/react-canvas-draw/pull/54

Reply:
ok